src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -314,141 +314,105 @@ inline double less_or_equal(double a, double b) {
     if (std::isnan(a) || std::isnan(b)) {
         res = false;
     } else if (std::isinf(b) && b > 0) {
-        // b is grater than any number or eq the +Inf
+        // b is greater than any number or eq the +Inf
         res = true;
     } else if (std::isinf(a) && a > 0) {
         res = false;
     } else {
-        res = (std::fabs(b - a) <= (std::fmax(std::fabs(a), std::fabs(b)) * eps) || a < b);
+        res = std::fabs(b - a) <= eps || a <= b;
```

Comment:
In general it's not guaranteed that `float(a) == float(b)` (or any other floating point type)

Reply:
Suggestions? we can use smt like: `(std::min(std::fabs(a), std::fabs(b)) / std::max(std::fabs(a), std::fabs(b))) >= (1.f - eps)` in case `a != 0` and `b != 0`. Also we can use half of mantissa bit len as eps here. Also we can use the same value to compare diff with eps